services/server/server.go

```diff
@@ -256,6 +256,10 @@ func (s *Server) Readiness(ctx context.Context, req *serverpb.ReadinessRequest)
 
 // CheckUpdates checks PMM Server updates availability.
 func (s *Server) CheckUpdates(ctx context.Context, req *serverpb.CheckUpdatesRequest) (*serverpb.CheckUpdatesResponse, error) {
+	s.envRW.RLock()
+	updatesDisabled := s.envDisableUpdates
```

Comment:
Why do we check updates and return info about latest version if we want to hide them?

Reply:
There are some tickets about checking for updates and letting users know about them, but not show update button because DISABLE_UPDATES is in effect. That would require API extension, but less changes on pmm-managed side – due to this change.